docs/source/technical_tutorials/search/cmr_search_in_r.ipynb

```diff
@@ -0,0 +1,469 @@
+{
```

Comment:
<p>explain more: "This allows use to use the python based maap-py library from R."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/456/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>